crates/octez/src/async/rollup.rs

```diff
@@ -257,4 +257,42 @@ mod test {
             Some(PathBuf::from("/tmp/kernel_debug.log"))
         );
     }
+
+    #[test]
+    fn serialize_config() {
+        let config = OctezRollupConfigBuilder::new(
+            Endpoint::localhost(1234),
+            PathBuf::from("/base_dir"),
+            SmartRollupHash::from_str("sr1PuFMgaRUN12rKQ3J2ae5psNtwCxPNmGNK").unwrap(),
+            "operator".to_owned(),
+            PathBuf::from("/tmp/boot_sector.hex"),
+        )
+        .set_kernel_debug_file(Path::new("/tmp/kernel_debug.log"))
+        .set_data_dir(RollupDataDir::TempWithPreImages {
+            preimages_dir: PathBuf::from("/tmp/pre_images"),
+        })
+        .build()
+        .unwrap();
+
+        let json = serde_json::to_value(config.clone()).unwrap();
+        assert_eq!(
+            json,
+            serde_json::json!({
+                "binary_path": "octez-smart-rollup-node",
+                "octez_client_base_dir": "/base_dir",
+                "octez_node_endpoint": "http://localhost:1234",
+                "pvm_kind": "wasm_2_0_0",
+                "data_dir": {
```

Comment:
This should show the path to the actual data directory used by the rollup node

Reply:
we discussed in slack that we could skip this for now as the actaul data direcotory path isn't avaiable until the rollup is spawned
Ticket: https://linear.app/tezos/issue/JSTZ-243/include-rollup-data-dir-in-config